draft-ietf-quic-recovery.md

```diff
@@ -518,14 +518,22 @@ or when Initial or Handshake keys are discarded. This ensures the PTO is always
 set based on the latest RTT information and for the last sent packet in the
 correct packet number space.
 
+When setting the PTO timer, the ApplicationData packet number space (Section
+4.1.1 of {{QUIC-TLS}}) MUST be ignored until the handshake completes. Not arming
+the PTO for ApplicationData prevents unnecessary transmissions on a PTO
+expiration, such as:
+
+* an endpoint sending probe packets before obtaining the keys to process an
+  acknowledgement,
+
+* a client sending 0-RTT probe packets before confirming that the server is able
```

Comment:
I believe we should remove this bullet point. See https://github.com/quicwg/base-drafts/pull/3874#discussion_r451515406 for the reasoning.

Reply:
Yeah my point in https://github.com/quicwg/base-drafts/pull/3874#discussion_r451557053 was that if the server rejects the 0-RTT, then that rejection acts as an alternative to ACK, and that therefore it might be worth mentioning.

But anyways I've applied MT's suggestion in daf4de0, I think it resolves your concern.